packages/kbn-user-profile-components/src/user_profiles_popover.tsx

```diff
@@ -7,9 +7,9 @@
  */
 
 import type { EuiPopoverProps, EuiContextMenuPanelProps } from '@elastic/eui';
-import type { FunctionComponent } from 'react';
+import { FunctionComponent } from 'react';
```

Comment:
Why is `FunctionComponent` no longer imported as a `type`? As far as I can see it's only used as a type in this file

Reply:
fixed